maquetado/scss/book-list.scss

```diff
@@ -0,0 +1,37 @@
+.book-list {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  max-height: 768px;
+  max-width: 1366px;
+  padding: 88px 215px 8px 215px;
+
+  .book-item-list {
+    align-items: flex-start;
+    box-shadow: 4px 4px 10px $black-russian;
+    display: flex;
+    flex-direction: column;
+    margin: 18px;
+    max-height: 300px;
+    max-width: 198px;
+    padding: 28px 28px 23px 28px;
+  
+    .book-image-mini{
```

Comment:
Add space between `.book-image-mini` and `{`

Reply:
Done